test/suites/drift/suite_test.go

```diff
@@ -377,8 +377,8 @@ var _ = Describe("Drift", func() {
 	})
 	It("should disrupt nodes that have drifted due to AMIs", func() {
 		// Choose an old static image (AL2023 AMIs don't exist for 1.22)
-		oldCustomAMI := env.GetAMIBySSMPath(lo.Ternary(env.K8sMinorVersion() == 23,
-			"/aws/service/eks/optimized-ami/1.23/amazon-linux-2023/x86_64/standard/amazon-eks-node-al2023-x86_64-standard-1.23-v20240307/image_id",
+		oldCustomAMI := env.GetAMIBySSMPath(lo.Ternary(env.K8sMinorVersion() == 25,
```

Comment:
This AMI exists for 1.23, so I think that we can actually entirely remove the ternary here

Reply:
Yeah, but don't we want this functionality for when 1.25 ends support?

Reply:
We can add it in then when that happens. The main reason this logic was in there right now was because there literally wasn't a 1.22 version of the AMI that was released. There is a 1.24 version, so going from one version to the next should be fine.